requirements.txt

```diff
@@ -16,5 +16,6 @@ scikit-learn
 singledispatchmethod
 tensorflow
 tensorflow-datasets>=4.1.0
+tensorflow-graphics
```

Comment:
I would prefer keeping tensorflow-graphics out of the main Kubric requirements for now. 
You can put it in `requirements_full.txt` instead to ensure that it will be in the docker image, but without making the pip-package too heavy.